### PR TITLE
test(tip, tip-group, tip-manager): add component token E2E tests

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #7180

## Summary

✨🧪✨

### CODE REVIEW

File: packages/calcite-components/src/components/tip/tip.scss [hunk 1]
```diff
@@ -88,7 +86,7 @@
 ::slotted(a) {
   @apply focus-base;
 
-  color: var(--calcite-tip-slotted-link-text-color, var(--calcite-color-brand));
+  color: var(--calcite-color-brand);
```
Comment (alisonailea) [inline packages/calcite-components/src/components/tip/tip.scss:89]: why did you remove `--calcite-tip-slotted-link-text-color`?

**Review by alisonailea — APPROVED** [review]
Looks good! One question

File: packages/calcite-components/src/components/tip/tip.scss (hunk 1 above)
Comment (jcfranco) [reply packages/calcite-components/src/components/tip/tip.scss:89]: IIRC, we added `--calcite-tip-slotted-link-text-color` initially in the component token update phase, but decided to not add component tokens for slotted elements in favor of having users override these directly.

**Comment by jcfranco** [comment]
@alisonailea Gonna go ahead and merge this, but LMK if we need to follow-up on https://github.com/Esri/calcite-design-system/pull/9589#discussion_r1640578769.
